lib/typescript/httpclient/payload/PaginatedPayload.ts

```diff
@@ -1,4 +1,4 @@
 export interface PaginatedPayload<T> {
   data: T[];
-  response_metadata: {previous_cursor: string; next_cursor: string; total: number};
+  pagination_data: {previous_cursor: string; next_cursor: string; total: number};
```

Comment:
I was waiting for this pr so we could fix the problems here. It's maybe a little annoying though. We'd need:

- change the casing so that it reflects JS conventions
- have a mapper in the library so that we can convert between two (even though... maybe there's already mappers and we can enrich them)

Reply:
ok! fixing now